controllers/phapplication_controller.go

```diff
@@ -85,7 +87,11 @@ func (r *PhApplicationReconciler) generateDeploymentSpec(phApplication *v1alpha1
 	}
 	deployment.Spec.Template.ObjectMeta.Labels = labels
 	deployment.Spec.Template.Spec = *podSpec
-	spawner, err := graphql.NewSpawnerForPhApplication(phApplication.AppID(), *groupInfo, *instanceTypeInfo, globalDatasets, *podSpec)
+	options := graphql.SpawnerOptions{
+		PhfsEnabled: r.PhfsEnabled,
+		PhfsPVC:     r.PhfsPVC,
+	}
+	spawner, err := graphql.NewSpawnerForPhApplication(phApplication.AppID(), *groupInfo, *instanceTypeInfo, globalDatasets, *podSpec, options)
```

Comment:
Since we have spawner options now, should we pass the `globalDatasets` in options as `additionalDatasets` to normalize the function parameters? WDYS?